guides/release/models/pushing-records-into-the-store.md

```diff
@@ -82,20 +85,23 @@ serializer before pushing it into the store, you can use the
 
 ```javascript {data-filename=app/serializers/album.js}
 import DS from 'ember-data';
+const { RESTSerializer } = DS;
 
-export default DS.RESTSerializer.extend({
+export default class AlbumSerializer extends RESTSerializer {
   normalize(typeHash, hash) {
     hash['songCount'] = hash['song_count']
     delete hash['song_count']
-    return this._super(typeHash, hash);
+    return super(typeHash, hash);
   }
-});
+}
 ```
 
 ```javascript {data-filename=app/routes/application.js}
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
-export default Route.extend({
+export default class ApplicationRoute extends Route {
+  @service store;
   model() {
     this.store.pushPayload({
```

Comment:
kind of odd that this model hook isn't returning anything. Wouldn't we want to return what we pushed?